### ISSUE TITLE
cloud.get_file doesn't return a full idelib.dataset

### ISSUE THREAD

**Issue by shanlyMIDE** [MideTechnology/endaq-python#162]
This function doesn't fully parse the file and internally should use the `ide.get_doc` function.

Here is the code we need to resolve: https://github.com/MideTechnology/endaq-python/blob/8b971e95e10bf1248618c7c7ee53b3e562b3502d/endaq/cloud/core.py#L102

**Comment by shanlyMIDE** [comment]
Now that we are in a shared library, this should be pretty easy to resolve

```python
  #Get the URL via the API to Download File
  download_url = requests.get(cloud_access.domain + "/api/v1/files/download/" + files.values[0],
                            headers={"x-api-key": cloud_access.api_key}).json()['url']

  #Download File into a idelib.dataset Object                            
  doc = endaq.ide.get_doc(download_url)       
```

**Comment by StokesMIDE** [comment]
To elaborate: The function currently instantiates a `idelib.dataset.Dataset` object directly. This creates the object but does not load the data. Addressed in PR #163, but needs testing.
